spec/ibandit/iban_spec.rb

```diff
@@ -1662,10 +1662,68 @@
     end
   end
 
-  describe "#valid?" do
+  describe "Pseudo IBAN #valid?" do
+    let(:country_code) { "CA" }
+    let(:arg) do
+      {
+        country_code: country_code,
+        bank_code: "0036",
+        branch_code: "00063",
+        account_number: "1234567",
+      }
+    end
+
     describe "validations called" do
       after { iban.valid? }
 
+      specify { expect(iban).to receive(:valid_pseudo_iban?).at_least(1) }
```

Comment:
Not exactly a fan of testing internal methods but if this is existing and we find it valuable then its fine